src/components/Dialogs/ImageCropDialog/ImageCropDialog.style.ts

```diff
@@ -68,13 +68,13 @@ export const CropContainer = styled.div<{ rounded?: boolean }>`
       top: 0;
       border-radius: ${({ rounded }) => (rounded ? '50%' : '0')};
 
-      box-shadow: inset 0 0 0 2px rgba(187, 231, 255, 0.48);
+      box-shadow: inset 0 0 0 2px rgba(256, 256, 256, 0.32);
```

Comment:
As far as I'm aware RGB values go up to `255` :P

On a more serious note, those semi-transparent layers have been define as part of the colors. Maybe to keep this tidy we should introduce those into `theme.colors`. Then we could use those both here and below for `.cropper-modal`

Reply:
You can find those here https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=4882%3A142